.github/workflows/build-with-bal-test-native.yml

```diff
@@ -0,0 +1,41 @@
+name: GraalVM Check
```

Comment:
Do we need both native and graalvm workflows?

Reply:
We don't need it. Removed with [2cc8a76](https://github.com/ballerina-platform/module-ballerinax-aws.dynamodbstreams/pull/2/commits/2cc8a76c20c0d3cee16f82892cc2b39ee6eea76b)